build.sbt

```diff
@@ -91,10 +104,14 @@ lazy val play24ConductRClientLib = project
   .dependsOn(testLib % "test->compile")
 
 
-// Test library
+// Test libraries
 lazy val testLib = project
```

Comment:
I'd rename the `testLib` to `scalaTestLib` to make it more obvious that the base test library uses Scala. Dependencies still stay the same.